src/VisualStudio/Core/Impl/SolutionExplorer/AnalyzerItem/AnalyzerItemSource.cs

```diff
@@ -2,237 +2,174 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
this is effectively a rewrite.  I recommend SxS diff view.

Reply:
This was rewritten in the style of https://github.com/dotnet/roslyn/pull/74444 and https://github.com/dotnet/roslyn/pull/74448